draft-ietf-cbor-7049bis.md

```diff
@@ -1206,6 +1206,10 @@ floats start as a 64-bit float, then do a test conversion to a 32-bit
 float; if the result is the same numeric value, use the shorter value
 and repeat the process with a test conversion to a 16-bit float. (This
 rule selects 16-bit float for positive and negative Infinity as well.)
+While IEEE floats can represent both positive and negative zero as
+distinct values, the application may not distinguish these and decide
```

Comment:
"might not distinguish these and might decide..."

Reply:
Agree; will fix